src/Compilers/VisualBasic/Test/Semantic/Compilation/CompilationAPITests.vb

```diff
@@ -1492,6 +1492,43 @@ BC2014: the value '_' is invalid for option 'RootNamespace'
             Assert.Equal(SymbolKind.NamedType, tupleWithNames.Kind)
         End Sub
 
+        <Fact()>
+        <WorkItem(36047, "https://github.com/dotnet/roslyn/issues/36047")>
+        Public Sub CreateArrayType_DefaultArgs()
+            Dim comp = DirectCast(VisualBasicCompilation.Create(""), Compilation)
+            Dim elementType = comp.GetSpecialType(SpecialType.System_Object)
+
+            Dim arrayType = comp.CreateArrayTypeSymbol(elementType)
```

Comment:
>CreateArrayTypeSymbol [](start = 33, length = 21)

Do we have a test for CreateArrayTypeSymbol with rank 0?

Reply:
The call below: `CreateArrayTypeSymbol(elementType, Nothing)`.

Added explicit calls with `0` as well.

---
In reply to: [302728651](https://github.com/dotnet/roslyn/pull/37048#discussion_r302728651) [](ancestors = 302728651)